packages/core/src/plugins/meta/discogs.test.ts

```diff
@@ -0,0 +1,19 @@
+import fetchMock from 'fetch-mock';
+
+describe('discogs album tracks test', () => {
+
+  beforeEach(() => {
+    fetchMock.reset();
+  });
+
+  it('search for albums', async () => {
```

Comment:
Sorry, this test won't do anything. You just set up a mock then run a loop on an empty array, which will not run any expects. The mock will not be consumed since there is nothing that could call it in this test.